HOWTO-make-a-release.md

```diff
@@ -0,0 +1,400 @@
+
+# Table of contents
+
+-   [Prerequisites](#prerequisites)
+    -   [Software](#software)
+    -   [Repositories](#repositories)
+    -   [PGP / GnuPG key](#pgp-gnupg-key)
+    -   [SSH access](#check-your-access)
+    -   [A method for reviewing](#a-way-to-reviewing)
+-   [Pre-publishing tasks](#pre-publishing-tasks)
+    -   [Freeze the source repository](#freeze-the-source-repository) [the day before release]
+    -   [Prepare your repository checkouts](#prepare-your-repository-checkouts)
+    -   [Make sure that the openssl source is up to date](#make-sure-that-the-openssl-source-is-up-to-date)
+    -   [Generate the tarball and announcement text](#generating-the-tarball-and-announcement-text)
+        -   [OpenSSL 3.0 and on](#openssl-3.0-and-on)
+        -   [OpenSSL before 3.0](#openssl-before-3.0)
+    -   [Update the website locally](#update-the-website-locally) [do not push]
+-   [Publish the release](#publish-the-release)
+-   [Post-publishing tasks](#post-publishing-tasks)
+    -   [Check the website](#check-the-website)
+    -   [Send the announcement mail](#send-the-announcement-mail)
+    -   [Send out the Security Advisory](#send-out-the-security-advisory)
+    -   [Unfreeze the source repository](#unfreeze-the-source-repository)
+    -   [Security fixes](#security-fixes)
+    -   [Keep in touch](#keep-in-touch)
+
+
+# Prerequisites
+
+## Software
+
+Apart from the basic operating system utilities, you must have the following
+programs in you `$PATH`:
+
+- openssl
+- ssh
+- gpg
+- git
+
+(note: this may not be a complete list)
+
+## Repositories
+
+You must have access to the following repositories:
+
+-   `openssl-git@git.openssl.org:openssl.git`
+
+    This is the usual main source repository
+
+-   `openssl-git@git.openssl.org:openssl-web.git`
+
+    This is the website repository
+
+-   `openssl-git@git.openssl.org:tools.git`
+
+    This contains certain common tools
+
+## PGP / GnuPG key
+
+You must have a PGP / GnuPG key, and its fingerprint should be present
+in the file `doc/fingerprints.txt` in the source of the immediately prior
+OpenSSL release.
+
+## SSH access
+
+To perform a release, you must have appropriate access to OpenSSL's
+development host, dev.openssl.org.  To test this, try to log in with
+ssh:
+
+    ssh dev.openssl.org
+
+You must also check that you can perform tasks as the user 'openssl'
+on dev.openssl.org.  When you have successfully logged in, test your
+access to that user with sudo:
+
+    sudo -u openssl id
+
+## A method for reviewing
+
+For reviewing to take place, the release person and the reviewer need a
+way to share changes that are being applied.  Most commonly, that's done
+as PRs (for normal releases) or security advisories (for undisclosed
+security fixes) through Github.
+
+Security advisories are created using the Github Security tab, and will
+generate a private repository, to which you can add collaborators (the
+reviewer, for instance), and use to fix the issue via pull requests.
```

Comment:
"use to fix the issue" doesn't quite sound right

Reply:
Is "use *it* to fix the issue" be better?  If not, I'd like some help with wording this.

Reply:
Yes, that's fine

Reply:
Done